File: solutions/swift/lasagna/1/Sources/Lasagna/Lasagna.swift
let expectedMinutesInOven = 40

func remainingMinutesInOven(elapsedMinutes: Int)->Int {
    return expectedMinutesInOven - elapsedMinutes
}

func preparationTimeInMinutes(layers: Int)->Int {
    return 2 * layers
}

func totalTimeInMinutes(layers: Int, elapsedMinutes: Int)->Int {
    return preparationTimeInMinutes(layers:layers) + elapsedMinutes
}

File: solutions/java/lasagna/1/src/main/java/Lasagna.java
public class Lasagna {
    public int expectedMinutesInOven(){
        return 40;
    }

    public int remainingMinutesInOven(int time){
        return expectedMinutesInOven()-time;
    }

    public int preparationTimeInMinutes(int layers){
        return 2*layers;
    }

    public int totalTimeInMinutes(int layers, int time){
        return preparationTimeInMinutes(layers)+time;
    }
}
